highLevelInterface/pwaNloptFit.cc

```diff
@@ -25,13 +25,42 @@ double
 rpwaNloptFunc(unsigned int n, const double* x, double* gradient, void* func_data)
 {
 	const pwaLikelihood<complex<double> >* L = (const pwaLikelihood<complex<double> >*)func_data;
-	if(n != L->nmbPars()) {
+	if(n != L->nmbPars() - L->nmbParsFixed()) {
 		printErr << "parameter mismatch between NLopt and pwaLikelihood. Aborting..." << endl;
+		printErr << "Likelihood needs "  << L->nmbPars() << " parameters including " << L->nmbParsFixed() << " fixed parameters. Got " << n << " parameters" << endl;
```

Comment:
Merge printErr message with the line above, i.e. "parameter mismatch between NLopt and pwaLikelihood: likelihood needs "  << ....  "Aborting..."

Reply:
Done